src/widget/openglwindow.cpp

```diff
@@ -12,6 +12,9 @@
 OpenGLWindow::OpenGLWindow(WGLWidget* pWidget)
         : m_pWidget(pWidget) {
     setFormat(WaveformWidgetFactory::getSurfaceFormat());
+    // Set the tooltip flag to prevent this window/widget from getting
+    // keyboard focus on click.
+    setFlag(Qt::ToolTip);
```

Comment:
```suggestion
#if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
    setFlag(Qt::ToolTip);
#else
    setFlag(Qt::Tool);
#endif
```

Reply:
Unfortunately this doesn't work either in Qt6.
Also tried _SplashScreen_ and _SubWindow_

I suggest to stick with this for 2.4.x and take a closer look at the qt sources, i.e. the Qt5 -> Qt6 changes.

Reply:
Strange. Adding setFlag(Qt::Tool) definitely works for me with main on macOS...

Reply:
Ha, obvious `Qt::WindowDoesNotAcceptFocus` flag does the trick on Linux wit Qt 6.2.3

Will open a fixup.